app/scaffold-controllers/navigationController.js

```diff
@@ -31,11 +42,14 @@ function(
         errorController,
         drawerEventHandler
     ) {
+        var layoutPromise = AnchoredLayoutPlugin.init();
```

Comment:
Is there a need to capture this in a variable? In the promise.join below, we didn't capture the `inits` in a vairable... I think we should go for consistency, whichever we choose :boom: